__tests__/index.test.ts

```diff
@@ -627,16 +637,115 @@ describe('cast', () => {
   })
 
   test('casts binary data to array of 8-bit unsigned integers', () => {
-    expect(cast({ name: 'test', type: 'BLOB' }, '')).toEqual(new Uint8Array([]))
-    expect(cast({ name: 'test', type: 'BLOB' }, 'Å')).toEqual(new Uint8Array([197]))
-    expect(cast({ name: 'test', type: 'VARBINARY' }, 'Å')).toEqual(new Uint8Array([197]))
+    expect(cast({ name: 'test', type: 'BLOB', charset: 63 }, '')).toEqual(new Uint8Array([]))
+    expect(cast({ name: 'test', type: 'BLOB', charset: 63 }, 'Å')).toEqual(new Uint8Array([197]))
+    expect(cast({ name: 'test', type: 'VARBINARY', charset: 63 }, 'Å')).toEqual(new Uint8Array([197]))
   })
 
   test('casts binary text data to text', () => {
-    expect(cast({ name: 'test', type: 'VARBINARY', flags: 4225 }, 'table')).toEqual('table')
+    expect(cast({ name: 'test', type: 'VARBINARY', charset: 255 }, 'table')).toEqual('table')
```

Comment:
Similarly in this case, a charset of 0 also isn't going to yield what's expected, and needed to become 255 to match reality.